tools/monitoring/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine
+
+ARG pkg=github.com/knative/test-infra/tools/monitoring
```

Comment:
also, dockerfile should be under test-infra/images

Reply:
Moved the dockerfile to test-infra/images and updated Makefiles to reflect the directory changes.